phalcon/mvc/model/repository.zep

```diff
@@ -0,0 +1,596 @@
+
+/*
+ +------------------------------------------------------------------------+
+ | Phalcon Framework                                                      |
+ +------------------------------------------------------------------------+
+ | Copyright (c) 2011-2016 Phalcon Team (https://phalconphp.com)          |
+ +------------------------------------------------------------------------+
+ | This source file is subject to the New BSD License that is bundled     |
+ | with this package in the file docs/LICENSE.txt.                        |
+ |                                                                        |
+ | If you did not receive a copy of the license and are unable to         |
+ | obtain it through the world-wide-web, please send an email             |
+ | to license@phalconphp.com so we can send you a copy immediately.       |
+ +------------------------------------------------------------------------+
+ | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
+ |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ +------------------------------------------------------------------------+
+ */
+
+namespace Phalcon\Mvc\Model;
+
+use Phalcon\Mvc\ModelInterface;
+
+/**
+ * Phalcon\Mvc\Model\Repository
+ */
+class Repository implements RepositoryInterface
+{
+
+	/**
+	 * @var string
+	 */
+	protected _modelClass;
+
+	protected _modelsManager;
```

Comment:
Oh okay, there is already `_modelsManager`, forgot what i wrote about `createBuilder` in repository then :D